src/opengl.c

```diff
@@ -388,13 +388,14 @@ static int gl_createWindow( unsigned int flags )
    /* Create the window. */
    gl_screen.window = SDL_CreateWindow( APPNAME,
          SDL_WINDOWPOS_CENTERED, SDL_WINDOWPOS_CENTERED,
-         SCREEN_W, SCREEN_H, flags | SDL_WINDOW_SHOWN | SDL_WINDOW_RESIZABLE );
+         800, 600, flags | SDL_WINDOW_SHOWN | SDL_WINDOW_RESIZABLE
```

Comment:
I've marked this PR as a draft because, even if there are no other issues, I'm not happy with this part.

The problem is the width and height passed here are in "screen coordinates", not pixels. So it ended up trying to create an 8k window (since these are multiplied by the scale factor), and generate a nebula at that resolution...

Reply:
Other than that, does it work well? It should be possible to define the nebula resolution somewhere else instead of just using the screen resolution.

Another issue is that font will probably look really bad unless we use some sort of higher resolution rendering.

Reply:
Yeah, it should work other than that.

> Another issue is that font will probably look really bad unless we use some sort of higher resolution rendering.

I think that's already a problem with the current font rendering, when a scale factor is set. I don't know if this really changes that.

My "plan" is to reimplement font rendering using [signed distance fields](https://steamcdn-a.akamaihd.net/apps/valve/2007/SIGGRAPH2007_AlphaTestedMagnification.pdf), which should scale well without performance issues while also allowing things like outlines to be added in the fragment shader, but I haven't made too much progress with that.